style.css

```diff
@@ -334,3 +334,77 @@ button {
   line-height: 20px;
   color: #253858;
 }
+
+.contact-form {
+  background-image: url(src/Contact-me.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contactContainer {
+  background-image: url(src/contact-shapes-mobile.png);
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: right;
+}
+
+.form-title {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  color: #fff;
+  text-align: center;
+}
+
+.content {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  color: #ebebff;
+  text-align: center;
+  margin: 10px;
+}
+
+#name,
+#email {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  border: 1px solid #ccc;
+  border-radius: 8px;
+  box-sizing: border-box;
+  padding: 12px;
+  margin-top: 10px;
+  margin-bottom: 8px;
+}
```

Comment:
- [ ] Kindly add `width: 90%;` here to be like the Figma design. And like the message input.